tests/parser.js

```diff
@@ -15,9 +15,16 @@ var existsSync = fs.existsSync || path.existsSync;
 var suite = new YUITest.TestSuite({
     name: 'Parser Test Suite',
     setUp: function () {
+        process.chdir(__dirname);
         var json = (new Y.YUIDoc({
             quiet: true,
-            paths: ['input/'],
+            paths: [
```

Comment:
These tests were naively presuming all fixtures were valid. I've narrowed the list to ones required for passing tests.